package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "circos": "git+https://github.com/matthewchan15/circosJS.git#matthewchan15-zoom-pan-svg",
     "fast-memoize": "^2.5.1",
-    "fornac": "^1.1.8",
+    "fornac": "git://github.com/plotly/fornac.git",
```

Comment:
Let's put a hash here, so if and when we upgrade it's intentional :)
Same for all the git-referenced packages in fact. Even the circos branch reference isn't unique, a commit hash is better (and while I'm looking at this... much as we appreciate @matthewchan15's work, circos should probably be moved into the plotly namespace as well)